Source/cmake/OptionsWPE.cmake

```
@@ -77,7 +77,11 @@ WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SHAREABLE_RESOURCE PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_THUNDER PRIVATE ${ENABLE_DEVELOPER_MODE})
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_TOUCH_EVENTS PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEB_RTC PRIVATE ${ENABLE_EXPERIMENTAL_FEATURES})
-WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_WEBXR PRIVATE ${ENABLE_EXPERIMENTAL_FEATURES})
+
+if (${ENABLE_EXPERIMENTAL_FEATURES} AND "${WPE_VERSION}" VERSION_GREATER_EQUAL "1.13.1")
```

I think you can write this more simply as:

```suggestion
if (ENABLE_EXPERIMENTAL_FEATURES AND ${WPE_VERSION} VERSION_GREATER_EQUAL 1.13.1)
```

That said, gating it behind ENABLE_EXPERIMENTAL_FEATURES is a big hammer to ensure that this functionality is not actually available to users. It's intended for half-baked features that users should not be able to enable without recompiling WebKit on their own. Is that really desired here? If it's more or less ready for users, sans bugs, then it's probably best to just enable it.

---

Building the gamepad support only depends on a recent-enough libwpe version. We should enable it by default, and packagers which have older versions of libwpe can always configure their builds with `-DENABLE_GAMEPAD=OFF`.

---

Sorry for the delay! I have updated it accordingly with your comments. Thanks @mcatanzaro